src/components/EditTodoComponent/editTasks.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {TextField, Checkbox} from "@material-ui/core";
+
+export default function EditTasks(props) {
+    if(props.editMode) {
+        return props.tasks.map((task, index) => {
+            return (
+            <div className="edit-box">
+                <Checkbox onChange={(event) => props.changeTaskChecked(!task.checked, index)} checked={task.checked}></Checkbox>
+                <TextField onChange={(event) => props.changeTaskText(event.target.value.trim(), index)} value={task.text}></TextField>
+            </div>
+        )
+        })
+        
+    } else {
+        return <div></div>
```

Comment:
Not return empty block. You can return null. It is more correct, because else create empty block with styles.